hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKey.java

```diff
@@ -67,14 +70,9 @@ public OzoneKey(String volumeName, String bucketName,
                   String keyName, long size, long creationTime,
                   long modificationTime, ReplicationType type,
                   int replicationFactor) {
-    this.volumeName = volumeName;
-    this.bucketName = bucketName;
-    this.name = keyName;
-    this.dataSize = size;
-    this.creationTime = Instant.ofEpochMilli(creationTime);
-    this.modificationTime = Instant.ofEpochMilli(modificationTime);
-    this.replicationConfig = ReplicationConfig.fromTypeAndFactor(type,
-            ReplicationFactor.valueOf(replicationFactor));
+    this(volumeName, bucketName, keyName, size, creationTime, modificationTime,
+            ReplicationConfig.fromTypeAndFactor(type,
```

Comment:
Setting the instance `this(creationTime, modificationTime ) `can remain initialized with `Instant.ofEpochMilli(creationTime)` and `Instant.ofEpochMilli(modificationTime) `respectively.

Reply:
@neils-dev can you please explain, I am not sure I understood your comment.

Reply:
Sure @kerneltime, the intialization for the creationTime, modification can retain its previous initialization prior to the change:
**_previous:_** 
```
this.creationTime = Instant.ofEpochMilli(creationTime);
    this.modificationTime = Instant.ofEpochMilli(modificationTime);
```
**_now:_**
line 73,
`this(volumeName, bucketName, keyName, size, `**Instant.ofEpochMilli(creationTime)**, **Instant.ofEpochMilli(modificationTime),**` ...)`

Reply:
The constructor called internally does the same, right?

Reply:
Yes!